code/modules/ZAS/Fire.dm

```diff
@@ -181,34 +133,46 @@ If it gains pressure too slowly, it may leak or just rupture instead of explodin
 			else
 				enemy_tile.adjacent_fire_act(loc, air_contents, air_contents.temperature, air_contents.volume)
 
+	if(firelevel > 6)
+		set_light(7, 3, no_update = TRUE)
```

Comment:
should probably be in update icon too, I tihnk that's where we put light changes too